build.assets/Dockerfile

```diff
@@ -40,11 +32,16 @@ RUN (mkdir -p /opt/protoc && \
      git clone https://github.com/gogo/protobuf --branch ${GOGO_PROTO_TAG} /src/github.com/gogo/protobuf && cd /src/github.com/gogo/protobuf && make install && \
      git clone https://github.com/grpc-ecosystem/grpc-gateway --branch ${GRPC_GATEWAY_TAG} /src/github.com/grpc-ecosystem/grpc-gateway && cd /src/github.com/grpc-ecosystem/grpc-gateway && pwd && go install ./protoc-gen-grpc-gateway)
 
-ENV PROTO_INCLUDE "/usr/local/include":"/src":"${GRPC_GATEWAY_ROOT}/third_party/googleapis":"${GOGOPROTO_ROOT}/gogoproto"
+ARG USER_ID=1000
```

Comment:
That's a good point. I believe 995 was chosen randomly. Because it is buildbox container you can build it with any user who has access to default GOMOD/GOPATH directories. 
Yes, for future improvements like mounting GOMOD directory from the host to cache builds it will be better to use the user from the host.
@a-palchikov if you have any thoughts why 996 UID was chosen I can change it back.